daemon/execdriver/windows/exec.go

```diff
@@ -0,0 +1,148 @@
+// +build windows
+
+package windows
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/docker/docker/daemon/execdriver"
+	"github.com/docker/docker/pkg/stringid"
+	"github.com/microsoft/hcsshim"
+	"github.com/natefinch/npipe"
+)
+
+func (d *driver) Exec(c *execdriver.Command, processConfig *execdriver.ProcessConfig, pipes *execdriver.Pipes, startCallback execdriver.StartCallback) (int, error) {
+
+	var inListen, outListen, errListen *npipe.PipeListener
+
+	active := d.activeContainers[c.ID]
+	if active == nil {
+		return -1, fmt.Errorf("Exec - No active container exists with ID %s", c.ID)
+	}
+
+	var (
+		term execdriver.Terminal
+		err  error
+	)
```

Comment:
since you have this `var` block already, any reason not to move the 4-5 var definitions here that are sprinkled throughout the rest of the function?